README.md

```diff
@@ -1,88 +1,56 @@
 
+## Download the latest release
 
+You can download the `jar`-file of the latest release version on our 
+[GitHub Release Page](https://github.com/fact-project/fact-tools/releases)
 
-Building the *fact-tools*
--------------------------
 
-The *fact-tools* are provided as a standard Maven project. They can be
-built by calling maven as follows:
+## Building the `FACT-Tools`
 
-     # mvn package
 
-This will automatically download the required libraries defined as dependencies
-in the project definition and will create a final bundled Jar file that
-includes all required classes.
+The `FACT-Tools` are provided as a standard Maven project.
+They can be built by calling maven as follows:
 
-Continous Integration
--------------------------
+```
+$ mvn package
+```
 
-A Travis instance is running. 
+This will automatically download the required libraries defined as dependencies in the project definition, run the unit test suite and will create a final bundled Jar file that includes all required classes in the `target` directory.
 
-[![Build Status](https://travis-ci.org/fact-project/fact-tools.svg?branch=master)](https://travis-ci.org/fact-project/fact-tools)
 
+## Running the `FACT-Tools`
 
+The final Jar file is called `fact-tools-VERSION.jar`,
+ where `VERSION` is the current version of the project.
 
-Running the *fact-tools*
-------------------------
-
-The final Jar file is called `fact-tools-VERSION.jar`, where `VERSION` is the
-current version of the project. Running the *fact-tools* requires a simple
-process graph definition as an XML file.
+Running the `FACT-Tools` requires a simple process graph definition as an XML file.
 
 With an XML file at hand the process graph can be started by issuing
 
-     # java -jar fact-tools-VERSION.jar /path/to/process.xml
-     
-     
-Versioning Scheme
----------------
-Starting with version 0.6.5 we all solemly swear to abide to the following versioning scheme. All who fail to follow these rules will be forced
-to buy a round of alcoholic beverages for everyone else.
-
-* The master branch only contains stable versions and no SNAPSHOTS (stable in this case mean we are comfortable with publishing results and sharing the program with other people)
-* Version numbers have to look like this:  0.X.YZ(-SNAPSHOT). Where X,Y,Z are natural numbers.
-* An xml file written for version 0.X has to work for all versions 0.X.YZ
-* Major version changes i.e from 0.X to 0.Z will be tagged for release
-* For new experimental features use a feature branch. See [this tutorial](http://git-scm.com/book/en/Git-Branching-Basic-Branching-and-Merging) for more information.
-
-You will receive weekly hugs and much respect from your fellow peers for writing unit tests and documentation. 
-Especially for adding content to our ever growing [project website](http://sfb876.tu-dortmund.de/FACT/) by editing
-the content under the src/site directory.
-  
-
-Orientation for Naming of Processors and Keynames
```

Comment:
I suggest to keep the naming conventions. I would also be fine to keep them in another place and just link that location

Reply:
The paragraph was not really helpful I think. I think this is better suited for Pull Request reviews.

Reply:
Fair enough, I agree to delete them from the README. Do we have this conventions written down somewhere, so you can refer to it in a PR review? Otherwise you always need an experienced person that remembers our conventions and reminds others of them.